bin/main.rb

```diff
@@ -70,23 +30,43 @@ def check_win(input)
   system 'cls'
   until board.valid
     puts board.create_board
-    puts "It's #{player1} turn"
+    puts "It's #{player1.name} turn"
     puts ''
     input = gets.chomp.to_i
     board.check_valid_move(input)
   end
-  board.check_win(input)
+  board.square[input - 1] = player1.letter
+  system 'cls'
+  system 'clear'
+  puts board.create_board
+  if board.draw
+    puts "It's a draw!"
+    break
```

Comment:
Let's improve the draw condition
I'm sure this is not a draw condition what I'm seeing on this image. It happens when Player1 is winning by filling the last available slot.

![notadraw](https://user-images.githubusercontent.com/22434660/112383863-aeb62600-8d13-11eb-9e4f-084255ba5950.jpg)

Hint: you can add one more condition in Line 42. By checking if player1 is winning this time or not.